pkg/model/dev.go

```diff
@@ -892,11 +921,17 @@ func (dev *Dev) ToTranslationRule(main *Dev, namespace, username string, reset b
 			}
 			rule.Args = append(rule.Args, "-s", fmt.Sprintf("%s:%s", filename, s.RemotePath))
 		}
-	} else if len(dev.Args.Values) > 0 {
-		rule.Args = dev.Args.Values
-	} else if len(dev.Command.Values) > 0 {
-		rule.Command = dev.Command.Values
-		rule.Args = []string{}
+	} else {
+		if main.PersistentVolumeAccessMode() != apiv1.ReadWriteMany {
```

Comment:
If the access mode supports `ReadWriteMany`, we don't need to force required affinities to deploy all pods in the same node